installer/docker-installer.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+# authors:
+# Tony Vattathil tonynv@amazon.com
```

Comment:
Not sure if this needs updating. There were already other contributors to this before I made this change: https://github.com/aws-quickstart/taskcat/commits/develop/installer/docker-install-master

Reply:
I think we can just remove the authors line for this file. I will make that update post merge